README.md

```diff
@@ -37,8 +28,13 @@ See `sample` for example application. For now it's using ionic showcase as its b
 - Now clone this repository (https://github.com/aerogear/voyager-android.git)
 - Run the application to send query and mutation to the server and displaying to the user.
 
-### To check offline part:
+## Try offline capabilities of Offix-Android
 
 1. Make a mutation by going offline.
 2. Then, come online and click on "FromDb" button and you will see the results.
 3. Also, refresh the application to see the updated cache.
+
+
+#### Here is a short demo of the offline capabilites where the client performs a mutation of updating the Task3 and when it comes online again, changes are refelected back in the application.
```

Comment:
Header's too long, please shorten it.
Have a look at the issue I opened #41 on how to write headers